src/containers/email-confirm-modal/EmailConfirmModal.styles.js

```diff
@@ -6,14 +6,16 @@ export const styles = {
     boxShadow: 'none',
     borderRadius: '8px'
   },
-  title: {
-    marginTop: '30px',
-    fontSize: '21px',
-    fontWeight: 'bold',
-    color: 'palette.grey.500'
-  },
   button: {
     size: 'large',
     mt: '32px'
+  },
+  titleWithDescription: {
+    title: {
+      typography: 'h6',
+      color: 'primary.900',
+      fontWeight: '600',
```

Comment:
on figma 500, dellete this, h6 alredy has 500

Reply:
+